angular-locale_en-cn.js

```diff
@@ -80,10 +80,10 @@ angular.module("ngLocale", [], ["$provide", function($provide) {
                 5,
                 6
             ],
-            "fullDate": "y\u5e74M\u6708d\u65e5EEEE",
-            "longDate": "y\u5e74M\u6708d\u65e5",
-            "medium": "y\u5e74M\u6708d\u65e5 ah:mm:ss",
-            "mediumDate": "y\u5e74M\u6708d\u65e5",
+            "fullDate": "EEEE, MMMM d, y",
+            "longDate": "MMMM d, y",
+            "medium": "MMM d, y h:mm:ss a",
+            "mediumDate": "MMM d, y",
             "mediumTime": "ah:mm:ss",
```

Comment:
Does anyone know if the new formats are correct for the fullDate, longDate, medium, and mediumDate? It looks to me as though the old formats were using year, month, day, not month, day, year. @janamaysaccesso, what do you think?